executor/executor.cc

```diff
@@ -170,7 +181,9 @@ static bool collide;
 uint32 completed;
 bool is_kernel_64_bit = true;
 
+#if !GOOS_windows
```

Comment:
Why? We have this above:
```
#if defined(__GNUC__)
#define SYSCALLAPI
#define NORETURN __attribute__((noreturn))
#define ALIGNED(N) __attribute__((aligned(N)))
#define PRINTF(fmt, args) __attribute__((format(printf, fmt, args)))
#else
// Assuming windows/cl.
#define SYSCALLAPI WINAPI
#define NORETURN __declspec(noreturn)
#define ALIGNED(N) __declspec(align(N))
#define PRINTF(fmt, args)
#endif
```

Reply:
Compiling says it's an illegal alignment value.

Reply:
Okay.
But then this is not the way to address this.
We have portability layer specifically to avoid ifdef's scattered through the code. And we have it implemented for windows. And we go and ifdef-out all mentions of ALIGNED for 1 OS.
Define ALIGNED on windows to nothing with a comment explaining why.